src/commonMain/kotlin/exchange.dydx.abacus/output/account/TradingRewards.kt

```diff
@@ -36,6 +39,56 @@ data class TradingRewards(
 ) {
     companion object {
         internal fun create(
+            existing: TradingRewards?,
```

Comment:
we shouldn't need to pass in `existing` here, we can just grab it from the `internalState` right?

Reply:
`internalState` contains the indexer data in this case, and `existing` is the output data type..  The data come from multiple sources processed at different processors, so it's easier to do the data aggregation at the output.